WordPress/src/main/java/org/wordpress/android/ui/ActivityLauncher.java

```diff
@@ -126,31 +126,34 @@ public static void showMainActivityAndSignupEpilogue(Activity activity, String n
      * @param site     the preselected site
      */
     public static void showSitePickerForResult(Activity activity, SiteModel site) {
-        Intent intent = createSitePickerIntent(activity, site);
+        Intent intent = createSitePickerIntent(activity, site, false);
```

Comment:
Even if we have currently only a 2 state discrimination here I would maybe use an enum (like `SitePickerMode`). Wdyt?

Reply:
You are right Riccardo, this will make the code more clear. I fixed it with 54adab18f52343bd8a095ad6b35565d2ed5976ef